modules/openapi-generator/src/main/resources/JavaSpring/bodyParams.mustache

```diff
@@ -1 +1 @@
-{{#isBodyParam}}{{>paramDoc}}{{#useBeanValidation}} @Valid{{>beanValidationBodyParams}}{{/useBeanValidation}} @RequestBody{{^required}}(required = false){{/required}} {{^reactive}}{{>optionalDataType}}{{/reactive}}{{#reactive}}{{^isArray}}Mono<{{{dataType}}}>{{/isArray}}{{#isArray}}Flux<{{{baseType}}}>{{/isArray}}{{/reactive}} {{paramName}}{{/isBodyParam}}
```

Comment:
I believe the current behaviour is intentional. If a parameter is not required, it may not be present. Therefore, it is represented as Optional. You can see this inside `optionalDataType.mustache` where it checks the required property and useOptional setting. In fact, if `useOptional` is disabled it should have the same effect as what you're doing here.

```mustache
{{#useOptional}}{{#required}}{{{dataType}}}{{/required}}{{^required}}Optional<{{#useBeanValidation}}{{>beanValidationCore}}{{/useBeanValidation}}{{{dataType}}}>{{/required}}{{/useOptional}}{{^useOptional}}{{{dataType}}}{{/useOptional}}
```

Note very last part - `{{^useOptional}}{{{dataType}}}{{/useOptional}}`

Can you provide a clear example in the PR description or in a unit test of why you believe this change should be made?